src/Client.php

```diff
@@ -246,7 +256,7 @@ public function downloadFile(
             $url .= '?access_token=' . $this->apiKey;
         }
 
-        $this->client->requestBody(QueryInterface::GET, $url, ['save_to' => $path]);
+        $this->client->requestJson(QueryInterface::GET, $url, ['save_to' => $path]);
```

Comment:
Klausimas: o kodėl čia pakeitei `requestBody` į `requestJson`? Ar iki šiol šitas metodas neteisingai veikė?

Reply:
Testai neveikdavo, nes grazindavo ne tai ko tikejosi. Netikrinau kaip buvo pre Guzzle 6, bet dabar sitas variantas veikia.
```
1) Isign\Gateway\Tests\Integration\ClientTest::testFileDownload
Error: Call to a member function getBody() on null

/Users/lieutenant_lt/projects/gateway-sdk-php/src/Http/GuzzleClientAdapter.php:50
/Users/lieutenant_lt/projects/gateway-sdk-php/src/Client.php:259
/Users/lieutenant_lt/projects/gateway-sdk-php/tests/Integration/ClientTest.php:15
```

Reply:
**Call to a member function on null** tikrai neišsispręstų vien dėl to, kad kvieti kitą member function. :) Kodėl `$this->client` buvo `null`? Gal buvai dar kažką pakeitęs, o gal tai buvo šiaip vienkartinis testo nutikimas?